src/SFML/Window/VideoMode.cpp

```diff
@@ -76,6 +76,13 @@ bool VideoMode::isValid() const
     return std::find(modes.begin(), modes.end(), *this) != modes.end();
 }
 
+////////////////////////////////////////////////////////////
+Vector2u VideoMode::getAspectRatio() const
+{
+    const unsigned int divisor = std::gcd(this->size.x, this->size.y);
```

Comment:
```suggestion
    const unsigned int divisor = std::gcd(size.x, size.y);
```
We don't use `this->`.